### ISSUE TITLE
Click on desktop notification doesn't open the message in the app

### ISSUE THREAD

**Issue by santherics** [zulip/zulip-desktop#842]
Ubuntu 16.04 64bit + Unity
Zulip app 4.0.1 installed from the repo

When the notication appears and I click (once or twice, doesn't matter) on it, the app doesn't maximize and doesn't show the message.

**Comment by akashnimare** [comment]
Yeah, as far as I know, this is not supported in the electron so will have to wait until they provide this feature. can you investigate more on this @muskankhedia?

**Comment by santherics** [comment]
This feature works on Ubuntu 18.04 and Mate Desktop

**Comment by akashnimare** [comment]
@santherics yeah, it's platform limitations.
